commodore/catalog.py

```diff
@@ -213,6 +235,9 @@ def update_catalog(cfg: Config, targets: Iterable[str], repo: GitRepo):
     if cfg.migration == Migration.KAP_029_030:
         click.echo(" > Smart diffing started... (this can take a while)")
         difftext, changed = repo.stage_all(diff_func=_kapitan_029_030_difffunc)
+    if cfg.migration == Migration.IGNORE_YAML_FORMATTING:
```

Comment:
I didn't see this yesterday evening, but this should be else if, to ensure that the three options (kapitan migration, yaml formatting migration, or regular diff) form a single logical block (it's not a bug per se, at the moment, since we still correctly execute the regular diff when neither migration is selected).

```suggestion
    elif cfg.migration == Migration.IGNORE_YAML_FORMATTING:
```

Reply:
🤦‍♀️